README.md

```diff
@@ -41,6 +41,29 @@ Once we have the first stable version, this notice will be removed.
 
 The current code is compatible with the [`standard` branch of the official MLKEM repository](https://github.com/pq-crystals/kyber/tree/standard).
 
+
+## Development
+
+### Environment Setup
+
+All the development and build dependencies are specified in [flake.nix](flake.nix). We recommend installing them using [nix](https://nixos.org/download/).
+
+- **Setup with nix**
+    - Running `nix develop` will execute a bash shell with the development environment specified in [flake.nix](flake.nix).
+    - Alternatively, you can enable `direnv` by using `direnv allow`, allowing it to handle the environment setup for you.
```

Comment:
Would I still do `nix develop` in this case?

Reply:
with `direnv`, you won't need to run `nix develop` every time. It will watch the changes of `flake.nix` and reload the shell accordingly.

But you would need to install `direnv` and add a shellhook to your shell. Maybe I should also add this to the README for clarity

Reply:
@potsrevennil Yes, that might be helpful!

Reply:
@potsrevennil Can you upen another PR for that?

Reply:
Ok, will do